cumulusci/core/config/ScratchOrgConfig.py

```diff
@@ -141,9 +141,13 @@ def email_address(self):
     def days(self):
         return self.config.setdefault("days", 1)
 
+    @property
+    def alive(self):
```

Comment:
Let's call it `active`, because that's the same terminology used in platform documentation about scratch orgs.

Reply:
That's fine.

Do you also want me to rename the (older) days_alive method or leave it?

Reply:
Leave it.

Reply:
@prescod I'm still waiting for you to rename this.

Reply:
https://github.com/SFDO-Tooling/CumulusCI/blob/cleanup_org_state_api/cumulusci/core/config/ScratchOrgConfig.py#L145